src/utils/DiscLogger.ts

```diff
@@ -0,0 +1,36 @@
+import { IDiscLoggerOptions } from "../typings";
+import { format } from "date-fns";
+
+enum Colors {
```

Comment:
Sort to this order
> Reset
> Red
> Yellow
> Green
> Blue

Reply:
Done in [`e2124bb`](https://github.com/zhycorp/disc-11/pull/594/commits/e2124bbe32e8374cd080c89e6f0ec70062230421)